dataactvalidator/config/appropFields.csv

```diff
@@ -1 +1 @@
deobligationsrecoveriesrefundsbytas_cpe,deobligations_recoveries_r_cpe,TRUE,float,21,
endingperiodofavailability,ending_period_of_availabil,FALSE,str,4,
grossoutlayamountbytas_cpe,gross_outlay_amount_by_tas_cpe,TRUE,float,21,
mainaccountcode,main_account_code,TRUE,str,4,TRUE
obligationsincurredtotalbytas_cpe,obligations_incurred_total_cpe,TRUE,float,21,
otherbudgetaryresourcesamount_cpe,other_budgetary_resources_cpe,FALSE,float,21,
spendingauthorityfromoffsettingcollectionsamounttotal_cpe,spending_authority_from_of_cpe,FALSE,float,21,
statusofbudgetaryresourcestotal_cpe,status_of_budgetary_resour_cpe,TRUE,float,21,
subaccountcode,sub_account_code,TRUE,str,3,TRUE
unobligatedbalance_cpe,unobligated_balance_cpe,TRUE,float,21,
\ No newline at end of file
+daimsname,fieldname,fieldname_short,required,data_type,field_length,padded_flag
AdjustmentsToUnobligatedBalanceBroughtForward_CPE,adjustmentstounobligatedbalancebroughtforward_cpe,adjustments_to_unobligated_cpe,TRUE,float,21,
AgencyIdentifier,agencyidentifier,agency_identifier,TRUE,str,3,TRUE
AllocationTransferAgencyIdentifier,allocationtransferagencyidentifier,allocation_transfer_agency,FALSE,str,3,TRUE
AvailabilityTypeCode,availabilitytypecode,availability_type_code,FALSE,str,1,
BeginningPeriodOfAvailability,beginningperiodofavailability,beginning_period_of_availa,FALSE,str,4,
BorrowingAuthorityAmountTotal_CPE,borrowingauthorityamounttotal_cpe,borrowing_authority_amount_cpe,FALSE,float,21,
BudgetAuthorityAppropriatedAmount_CPE,budgetauthorityappropriatedamount_cpe,budget_authority_appropria_cpe,TRUE,float,21,
TotalBudgetaryResources_CPE,totalbudgetaryresources_cpe,total_budgetary_resources_cpe,TRUE,float,21,
BudgetAuthorityUnobligatedBalanceBroughtForward_FYB,budgetauthorityunobligatedbalancebroughtforward_fyb,budget_authority_unobligat_fyb,FALSE,float,21,
ContractAuthorityAmountTotal_CPE,contractauthorityamounttotal_cpe,contract_authority_amount_cpe,FALSE,float,21,
DeobligationsRecoveriesRefundsByTAS_CPE,deobligationsrecoveriesrefundsbytas_cpe,deobligations_recoveries_r_cpe,TRUE,float,21,
EndingPeriodOfAvailability,endingperiodofavailability,ending_period_of_availabil,FALSE,str,4,
GrossOutlayAmountByTAS_CPE,grossoutlayamountbytas_cpe,gross_outlay_amount_by_tas_cpe,TRUE,float,21,
MainAccountCode,mainaccountcode,main_account_code,TRUE,str,4,TRUE
ObligationsIncurredTotalByTAS_CPE,obligationsincurredtotalbytas_cpe,obligations_incurred_total_cpe,TRUE,float,21,
OtherBudgetaryResourcesAmount_CPE,otherbudgetaryresourcesamount_cpe,other_budgetary_resources_cpe,FALSE,float,21,
SpendingAuthorityfromOffsettingCollectionsAmountTotal_CPE,spendingauthorityfromoffsettingcollectionsamounttotal_cpe,spending_authority_from_of_cpe,FALSE,float,21,
StatusOfBudgetaryResourcesTotal_CPE,statusofbudgetaryresourcestotal_cpe,status_of_budgetary_resour_cpe,TRUE,float,21,
SubAccountCode,subaccountcode,sub_account_code,TRUE,str,3,TRUE
UnobligatedBalance_CPE,unobligatedbalance_cpe,unobligated_balance_cpe,TRUE,float,21,
```

Comment:
Can we make this not one line somehow? Last time it was a matter of updating the line separators in PyCharm